conf/deployment/vsphere/upi_1az_rhcos_lowerreq_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,16 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
+  worker_num_cpus: '8'
+  master_num_cpus: '8'
```

Comment:
are we planning to increase master CPUs because in vsphere config we use 4 cpu https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w.yaml

Reply:
You are right, will revert each configuration `master_num_cpus` to 4